Index: .idea/Peyton's Portfolio.iml
===================================================================
diff --git a/.idea/Peyton's Portfolio.iml b/.idea/Peyton's Portfolio.iml
deleted file mode 100644
--- a/.idea/Peyton's Portfolio.iml	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
+++ /dev/null	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="WEB_MODULE" version="4">
-  <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$">
-      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
-      <excludeFolder url="file://$MODULE_DIR$/temp" />
-      <excludeFolder url="file://$MODULE_DIR$/tmp" />
-    </content>
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
+++ /dev/null	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
Index: .idea/modules.xml
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
deleted file mode 100644
--- a/.idea/modules.xml	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
+++ /dev/null	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/Peyton's Portfolio.iml" filepath="$PROJECT_DIR$/.idea/Peyton's Portfolio.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
Index: Readme.txt
===================================================================
diff --git a/Readme.txt b/Readme.txt
deleted file mode 100644
--- a/Readme.txt	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
+++ /dev/null	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
@@ -1,1 +0,0 @@
-
Index: .idea/workspace.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/workspace.xml b/.idea/workspace.xml
new file mode 100644
--- /dev/null	(date 1741969625896)
+++ b/.idea/workspace.xml	(date 1741969625896)
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PropertiesComponent">{}</component>
+</project>
\ No newline at end of file
Index: package.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"name\": \"Peyton's Portfolio\",\r\n  \"version\": \"1.0.0\",\r\n  \"description\": \"\",\r\n  \"main\": \"index.js\",\r\n  \"scripts\": {\r\n    \"test\": \"echo \\\"Error: no test specified\\\" && exit 1\"\r\n  },\r\n  \"private\": true\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/package.json b/package.json
--- a/package.json	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
+++ b/package.json	(date 1741969628024)
@@ -1,5 +1,5 @@
 {
-  "name": "Peyton's Portfolio",
+  "name": "Peyton-Portfolio",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"VcsDirectoryMappings\">\r\n    <mapping directory=\"$PROJECT_DIR$\" vcs=\"Git\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
--- a/.idea/vcs.xml	(revision 7a1f8ae1644d19aff0da8d77d0c5ead1f9c4f9e5)
+++ b/.idea/vcs.xml	(date 1741969625896)
@@ -1,6 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-  </component>
+  <component name="VcsDirectoryMappings" defaultProject="true" />
 </project>
\ No newline at end of file
